### PR TITLE
Style events

### PR DESCRIPTION
**Pull request by seann1**
Closes issue #150

### CODE REVIEW

**Comment by seann1** [comment]
eventList1
eventList1

**Comment by seann1** [comment]
eventList1
eventList1

**Comment by seann1** [comment]
eventList2

**Comment by seann1** [comment]
eventList2

File: frontend/src/components/project_page/pages/ProjectLandingPage/components/ProjectEventList.tsx [hunk 1]
```diff
@@ -37,11 +40,62 @@ const ProjectEventList: FC<ProjectEventListProps> = ({ projectId }) => {
...
     <>
-      {events.map((event) => (
-        <div key={event.id} className="my-1">
-          <ProjectEventPreview event={event} now={new Date()} />
+      <div
+        className={`relative ${
+          !expanded ? "h-32" : "h-96"
+        } border-[1px] border-black box-content rounded shadow transition-height duration-500 ease-in-out`}
+      >
+        {/* {!expanded && ()} */}
```
Comment (mtso) [inline frontend/src/components/project_page/pages/ProjectLandingPage/components/ProjectEventList.tsx:48]: nit: prefer to delete dead code

File: frontend/src/components/project_page/pages/ProjectLandingPage/components/ProjectEventList.tsx [hunk 2]
```diff
@@ -37,11 +40,62 @@ const ProjectEventList: FC<ProjectEventListProps> = ({ projectId }) => {
...
+          className={`${
+            !expanded ? "h-32 overflow-hidden" : "h-96 overflow-y-scroll"
+          } transition-height duration-500 ease-in-out`}
+        >
+          {events.map((event) => (
+            <div key={event.id} className="my-1 ml-2">
+              <ProjectEventPreview event={event} now={new Date()} />
+            </div>
+          ))}
+          {/* {(() => {
```
Comment (mtso) [inline frontend/src/components/project_page/pages/ProjectLandingPage/components/ProjectEventList.tsx:84]: nit: prefer to remove dead code

**Review by mtso — APPROVED** [review]

**Pull request merged**